tests/dummy/app/templates/migration-guides.hbs

```diff
@@ -67,7 +67,7 @@
       <p>
         Version 0.3.0 includes major column resizing changes, but with
         <code>canAutoResize</code> defaulting to <code>true</code>.  We
-        recommend skipping this version and upgrading directly to 0.4.0; see
+        recommend skipping this version and upgrading direcly to 0.4.0; see
```

Comment:
You use vim right? You probably just hit x by mistake.
On May 20, 2015 11:45 PM, "Alex Zirbel" <notifications@github.com> wrote:

> In tests/dummy/app/templates/migration-guides.hbs
> <https://github.com/Addepar/ember-table/pull/309#discussion_r30776469>:
>
> > @@ -67,7 +67,7 @@
> >        <p>
> >          Version 0.3.0 includes major column resizing changes, but with
> >          <code>canAutoResize</code> defaulting to <code>true</code>.  We
> > -        recommend skipping this version and upgrading directly to 0.4.0; see
> > +        recommend skipping this version and upgrading direcly to 0.4.0; see
>
> I ... don't know how I managed this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Addepar/ember-table/pull/309/files#r30776469>.
>